processors/games/Duels.js

```diff
@@ -2,11 +2,965 @@
 * Duels
  */
 module.exports = ({
+  active_auras = 0,
+  active_cage = 0,
+  active_cosmetictitle = 0,
+  active_emblem = 0,
+  active_killmessages = 0,
+  active_projectile_trail = 0,
+  active_victory_dance = 0,
+  active_weaponpacks = 0,
+  all_modes_gold_title_prestige = 0,
+  all_modes_iron_title_prestige = 0,
+  amount_healed_standard = 0,
+  amount_healed = 0,
+  arcanist_kit_wins = 0,
+  armorer_kit_wins = 0,
+  arrows_from_rend_standard = 0,
+  arrows_from_rend = 0,
+  assassin_kit_wins = 0,
+  assassin_master_alechmy_hearts_standard = 0,
+  assassin_master_alechmy_hearts = 0,
+  assassin_strikes_from_cloak_standard = 0,
+  assassin_strikes_from_cloak = 0,
+  astronaut_kit_wins = 0,
+  best_blitz_winstreak = 0,
+  best_bridge_winstreak = 0,
+  best_classic_winstreak = 0,
+  best_mega_walls_winstreak = 0,
+  best_op_winstreak = 0,
+  best_overall_winstreak = 0,
+  best_skywars_winstreak = 0,
+  best_sumo_winstreak = 0,
+  best_uhc_winstreak = 0,
+  best_winstreak_mode_blitz_duel = 0,
+  best_winstreak_mode_bridge_doubles = 0,
+  best_winstreak_mode_bridge_duel = 0,
+  best_winstreak_mode_bridge_four = 0,
+  best_winstreak_mode_classic_duel = 0,
+  best_winstreak_mode_mw_duel = 0,
+  best_winstreak_mode_op_duel = 0,
+  best_winstreak_mode_ranked_1 = 0,
+  best_winstreak_mode_sumo_duel = 0,
+  best_winstreak_mode_sw_doubles = 0,
+  best_winstreak_mode_sw_duel = 0,
+  best_winstreak_mode_uhc_doubles = 0,
+  best_winstreak_mode_uhc_duel = 0,
+  best_winstreak_mode_uhc_four = 0,
+  best_winstreak_mode_uhc_meetup = 0,
+  blaze_amount_healed_standard = 0,
+  blaze_amount_healed = 0,
+  blaze_g_amount_healed_standard = 0,
+  blaze_g_amount_healed = 0,
+  blitz_duel_armorer_kit_wins = 0,
+  blitz_duel_astronaut_kit_wins = 0,
+  blitz_duel_blocks_placed = 0,
+  blitz_duel_bow_hits = 0,
+  blitz_duel_bow_shots = 0,
+  blitz_duel_creepertamer_kit_wins = 0,
+  blitz_duel_damage_dealt = 0,
+  blitz_duel_deaths = 0,
+  blitz_duel_donkeytamer_kit_wins = 0,
+  blitz_duel_farmer_kit_wins = 0,
+  blitz_duel_golem_kit_wins = 0,
+  blitz_duel_health_regenerated = 0,
+  blitz_duel_horsetamer_kit_wins = 0,
+  blitz_duel_hunter_kit_wins = 0,
+  'blitz_duel_hype train_kit_wins' = 0,
+  blitz_duel_kills = 0,
+  blitz_duel_kit_wins = 0,
+  blitz_duel_layout_paladin_5 = 0,
+  blitz_duel_losses = 0,
+  blitz_duel_melee_hits = 0,
+  blitz_duel_melee_swings = 0,
+  blitz_duel_paladin_kit_wins = 0,
+  blitz_duel_ranger_kit_wins = 0,
+  blitz_duel_rounds_played = 0,
+  blitz_duel_scout_kit_wins = 0,
+  'blitz_duel_shadow knight_kit_wins' = 0,
```

Comment:
Looks like I gave you the wrong advice...

This needs to be changed to 
```js
  'blitz_duel_shadow knight_kit_wins': blitz_duel_shadow_knight_kit_wins = 0,
```
for it to work.

Also you don't need to make a new PR for every change, you can just push to the same branch and it'll be automatically added.